src/Service/ServiceUtils.cpp

```diff
@@ -40,11 +39,10 @@ namespace orbit_service::utils {
 namespace fs = std::filesystem;
 
 using orbit_grpc_protos::TracepointInfo;
-using ::orbit_object_utils::CoffFile;
-using ::orbit_object_utils::CreateElfFile;
 using ::orbit_object_utils::CreateObjectFile;
-using ::orbit_object_utils::ElfFile;
+using ::orbit_object_utils::CreateSymbolsFile;
```

Comment:
is there a reason for using leading `::`?

Reply:
no, done